src/managers/RoleManager.js

```diff
@@ -115,7 +115,7 @@ class RoleManager extends BaseManager {
   create(options = {}) {
     let { name, color, hoist, permissions, position, mentionable, reason } = options;
     if (color) color = resolveColor(color);
-    if (permissions) permissions = Permissions.resolve(permissions).toString();
+    if (typeof permissions !== 'undefined') permissions = Permissions.resolve(permissions).toString();
```

Comment:
```suggestion
    permissions = Permissions.resolve(permissions).toString();
```
Don't see why we can't just do this actually. `Permissions.resolve` returns 0 permissions when passing undefined